docs/dev/api/api-testing.md

```diff
@@ -489,6 +489,12 @@ Runs a single test on Sauce Labs API Testing, creates a Log in your Dashboard, a
      <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>The test ID. To find it, go to your project > <strong>Tests</strong> tab > Hover over your test and click <strong>Edit</strong> > Grab the test ID from your browser's URL (<code>https://app.saucelabs.com/api-testing/project/&#123;projectId&#125;/test/<strong>&#123;testId&#125;</strong>/compose</code>).</p></td>
     </tr>
   </tbody>
+  <tbody>
+    <tr>
+     <td><code>buildId</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If this execution is associated to a build, the ID of the build should be provided, so that all resources can be associated to it.</p></td>
```

Comment:
@LoryApiFortress @theirish81 @lauramthomas 

- We should specify where the user can find their `buildId` and/or how to create one
- Can you also clarify what you mean by resources?

```suggestion
     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If your test execution is associated with a build, you'll need to provide your <a href="/api-testing/integrations/apifctl-cicd-integration/#-b-build-id">build ID</a>, so that all resources(??) can be associated with it.</p></td>
```

Reply:
@kimsauce the buildId is not retrieved in the platform, but provided by the system running the test.
As in:
1. The user's CI/CD platform runs a build. As part of this process a build ID is automatically generated in the scope of the pipeline (this is common to all CI/CD platforms)
2. We suggest you pass it over when you use apifctl, so that APIT can aggregate the data based on the build ID

Reply:
@lauramthomas @LoryApiFortress @theirish81 , maybe something like this then? 

```suggestion
     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If your test execution is associated with a CI/CD <a href="/api-testing/project-dashboard/#test-build-reports">build</a>, you'll need to provide your <a href="/api-testing/integrations/apifctl-cicd-integration/#-b-build-id">build ID</a> so that its associated data will be aggregated in your API Testing.</p></td>
```